kpatch-build/kpatch-build

```diff
@@ -773,16 +773,25 @@ for i in $FILES; do
 	mkdir -p "output/$(dirname "$i")"
 	cd "$SRCDIR" || die
 	find_kobj "$i"
-	if [[ "$KOBJFILE" = vmlinux ]]; then
-		KOBJFILE="$VMLINUX"
-	else
-		KOBJFILE="$TEMPDIR/module/$KOBJFILE"
-	fi
 	cd "$TEMPDIR" || die
 	if [[ -e "orig/$i" ]]; then
-		# create-diff-object orig.o patched.o kernel-object output.o Module.symvers patch-mod-name
-		"$TOOLSDIR"/create-diff-object "orig/$i" "patched/$i" "$KOBJFILE" \
-			"output/$i" "$SRCDIR/Module.symvers" "${MODNAME//-/_}" 2>&1 | logger 1
+		if [[ "$KOBJFILE" = vmlinux ]]; then
+			KOBJFILE_NAME=vmlinux
+			KOBJFILE_PATH="$VMLINUX"
+		else
+			KOBJFILE_NAME=$(basename "${KOBJFILE%.ko}")
+			KOBJFILE_NAME="${KOBJFILE_NAME/-/_}"
+			KOBJFILE_PATH="${TEMPDIR}/module/$KOBJFILE"
+		fi
+
+		SYMTAB="${TEMPDIR}/${KOBJFILE_NAME}.symtab"
```

Comment:
This could overwrite another symtab if there are two modules being patched which have the same name (but are in different directories).  Wouldn't break anything, but could make debugging confusing.

Maybe set SYMTAB conditionally.  In the vmlinux case it could be `$TEMPDIR/vmlinux.symtab`, and in the module case it could be `$KOBJFILE.symtab`.